xml/adm_shell.xml

```diff
@@ -473,7 +473,7 @@ echo "Hello World" <co xml:id="co-adm-shell-echo"/></screen>
 
   <variablelist>
    <varlistentry>
-    <term><literal>Command &gt; File</literal>
```

Comment:
Changing from uppercase to lowercase within `<literal/>` tag might be critical (although in this case , it would be better to write the commands in lowercase anyway). Or use `<replaceable/>` (did not check the entire file but I think those are example commands).

Reply:
That is fixed.